{"uuid": "a40ba1df-7f55-4a7f-84f7-95cdeeaff8b1", "children": ["3d688750-9bc0-4d05-ab88-0b8de2f8fb6f"], "befores": [{"name": "take_screenshot_on_failure", "status": "passed", "start": 1728981343693, "stop": 1728981343693}], "afters": [{"name": "take_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\KUNAL\\SekelTechAssignments\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\KUNAL\\SekelTechAssignments\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\KUNAL\\SekelTechAssignments\\conftest.py\", line 34, in take_screenshot_on_failure\n    if request.node.rep_call.failed:  # Check if the test failed\n"}, "start": 1728981346297, "stop": 1728981346300}], "start": 1728981343693, "stop": 1728981346300}